src/com/lftechnology/java/training/srijan/FirstTutorial.java

```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.java.training.srijan;
+
+import java.util.Scanner;
+
+public class FirstTutorial {
```

Comment:
Please refer the below link for writing java docs
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html